<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.3">
    <title>Links for pyexafs</title>
  </head>
  <body>
    <h1>Links for pyexafs</h1>
<a href="https://files.pythonhosted.org/packages/53/c0/7eccbd4be89d8b2d1043711f3ef783121c45435a4e0bb092529ad7cd565a/pyexafs-1.0.0b1-py3-none-any.whl#sha256=df6bff283fa9b2bc6f57b5f4e92f95eb05b020bb6fbf15001337172a52131916" data-yanked="Had an unfortunate error after uploading" data-dist-info-metadata="sha256=c387366c85ff41192aaa3c2836e48660e8bb5377f6eb85500b145d3729424a69" data-core-metadata="sha256=c387366c85ff41192aaa3c2836e48660e8bb5377f6eb85500b145d3729424a69">pyexafs-1.0.0b1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/d0/4f/336917dc71038fee6fc54fa0077bd038e06b4bbff19a988e4be9d4279bb8/pyexafs-1.0.0b1.tar.gz#sha256=9f929fbf037eec987a4b9516f3859163e9f56a9557e69e6abcd266a0683d2f2b" data-yanked="Had an unfortunate error after uploading" >pyexafs-1.0.0b1.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/bc/36/373fead9e25463b4757d991d484d2a80a6f948ebf78f9950718ad2f79c3c/pyexafs-1.0.0b2-py3-none-any.whl#sha256=eda306365082de3fb22b8889c5dbcc3d7df062ed3788e0f04fb7d8f8e0b725b4" data-yanked="Had an unfortunate error after uploading (again)" data-dist-info-metadata="sha256=7d3cce3723f1a5b2d0f15770e8d96dce7fcef75f7e5478748825601fa99f0c32" data-core-metadata="sha256=7d3cce3723f1a5b2d0f15770e8d96dce7fcef75f7e5478748825601fa99f0c32">pyexafs-1.0.0b2-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/b8/1b/fb8e10f29443b7fbd3d1d2bfb5743d74ff7ea278de01c1f463775b0201c8/pyexafs-1.0.0b2.tar.gz#sha256=fe7fa8dbb424bd8d62e411a9936a7c4cdb7d5e859a900320a5dab9ab86dded72" data-yanked="Had an unfortunate error after uploading (again)" >pyexafs-1.0.0b2.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/c3/8c/4752b708b9eec4acb13911ccf878df9a60232feec8982d773d525b51f344/pyexafs-1.0.0b3-py3-none-any.whl#sha256=27cfa3031bbdc8439bc6b35684bdf373adbed4e8f567f6952ff165d440b859eb" data-yanked=" Had an unfortunate error after uploading (still)" data-dist-info-metadata="sha256=7d58f85a5224e5cc9d73e1134ea183e37f30aa861fe6dcf9a05c7f64f7de412d" data-core-metadata="sha256=7d58f85a5224e5cc9d73e1134ea183e37f30aa861fe6dcf9a05c7f64f7de412d">pyexafs-1.0.0b3-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/c8/c3/8aa0ffc7282c36b89042e2e448863f79a3b739db96c98467f293e490df3d/pyexafs-1.0.0b3.tar.gz#sha256=4e039e752c22dca477947901cb419964cdd74ffaf941be7310de4b9dacc143fe" data-yanked=" Had an unfortunate error after uploading (still)" >pyexafs-1.0.0b3.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/2f/73/8e3b50c9cda5d904baf2b163d35f8f69db94044d2fa5a994f0b6b8d89206/pyexafs-1.0.0b4-py3-none-any.whl#sha256=bcfbb33755ffd7a4320cdb642e11c287ab5a04eb23ce93359392b86bf1a8f223" data-yanked="Had an unfortunate error after uploading (still) " data-dist-info-metadata="sha256=daaeda96d81206763772477b75ce1b47f828dd73412900f7b7007cfe31899bfa" data-core-metadata="sha256=daaeda96d81206763772477b75ce1b47f828dd73412900f7b7007cfe31899bfa">pyexafs-1.0.0b4-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/f1/7b/a7b1236ebd16ab608af100893a27bce6503045d272af8c91396f917092d5/pyexafs-1.0.0b4.tar.gz#sha256=da0b7c8f864044d63241faf8315058ce3a2c6401cf401781672d27264bd8286e" data-yanked="Had an unfortunate error after uploading (still) " >pyexafs-1.0.0b4.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/32/88/823b5eb0902c453f362a6f5c8e910180bb2770cd14939ab43a92ae712150/pyexafs-1.0.0b5-py3-none-any.whl#sha256=6e11dcdf3bd2caeb852835a46b46f13095fb1c6fceb71652eea26bb29f5ef9f0" data-yanked="Had an unfortunate error after uploading (still) " data-dist-info-metadata="sha256=8790909777f7921ce52cba80aa1f11ba39009f554a6d622cc4f8e40d8c41f8c3" data-core-metadata="sha256=8790909777f7921ce52cba80aa1f11ba39009f554a6d622cc4f8e40d8c41f8c3">pyexafs-1.0.0b5-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/cd/c8/eae8439e292a43b28343ff2b1f0028516db5a6017652206bda4283ebb1ec/pyexafs-1.0.0b5.tar.gz#sha256=55ead3d9a01fae545f185c7c6fc01acaa0da68fb7505628187d5309c7325d729" data-yanked="Had an unfortunate error after uploading (still) " >pyexafs-1.0.0b5.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/3e/12/71bc4d34051dbc55a0b70b74b6c4f357010cfff8b7e7e4ba3402e69f6139/pyexafs-1.0.0b6-py3-none-any.whl#sha256=e011d5d5442341af2419d6dd44a3da3b7aea15ebdcb7c747e109a338ba33b8ac" data-yanked="error..." data-dist-info-metadata="sha256=c92fb0a51e2aa64ffea58c92cce8a259f2f8fbc5a6c17cf44f359051aa49ceda" data-core-metadata="sha256=c92fb0a51e2aa64ffea58c92cce8a259f2f8fbc5a6c17cf44f359051aa49ceda">pyexafs-1.0.0b6-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/53/e6/c98cb6c309309957caa0e58deebf5692e1d548a07759958d74f02eea197f/pyexafs-1.0.0b6.tar.gz#sha256=50011ad3bb397045b168740b847637d280a5a34151b3934a04f09babefcbf941" data-yanked="error..." >pyexafs-1.0.0b6.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/ff/a7/6755b0f1f3a331571f4d22adee8467626edc278fc1e731626e3b09ba3cb3/pyexafs-1.0.0b7-py3-none-any.whl#sha256=fad7d4c0de7d2ca3fee126b5139ed70b30d93ed6c049f3e7fec22b7cc68a8f08" data-yanked="..." data-dist-info-metadata="sha256=dc0a16ea1b47c359031cc8e945c0bbc4bfd88bc9694bb5e32880bdbf4649007b" data-core-metadata="sha256=dc0a16ea1b47c359031cc8e945c0bbc4bfd88bc9694bb5e32880bdbf4649007b">pyexafs-1.0.0b7-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/63/b6/0a2a33830d9fd155be3b546398304fd6f2f059721e6de6779071e2124a6c/pyexafs-1.0.0b7.tar.gz#sha256=b1ce18f7272b0b9dfc7254c08254a0cf67054f21d84acad77b88413afbe7a2df" data-yanked="..." >pyexafs-1.0.0b7.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/d6/00/9f9265009c21ff61bcbb2049bf61441610f734d4ba2668c8c910155bdd31/pyexafs-1.0.0b8-py3-none-any.whl#sha256=f2a7ee0f75b65217acee24c76586fb2046b88e59411e960a1f8e60ad4ab4d54a" data-yanked="faulty" data-dist-info-metadata="sha256=2ab1e0781b6c772828307e8e8d3d0861e99057cf3de43e4da7d3d14c1ef30500" data-core-metadata="sha256=2ab1e0781b6c772828307e8e8d3d0861e99057cf3de43e4da7d3d14c1ef30500">pyexafs-1.0.0b8-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/1a/ba/adc96b5d2259195c15f79a94726fdaf489be517be7dd5bba5d0504af6831/pyexafs-1.0.0b8.tar.gz#sha256=d30a8225ef0cc8c2549f1e8c3dec77b496993cc1c24ae7edda27fcafd9c1e499" data-yanked="faulty" >pyexafs-1.0.0b8.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/24/90/2b0ab14249133027ad0166e598919991112e8e84b8ca13fc8672ad993c8a/pyexafs-1.0.0b9-py3-none-any.whl#sha256=fccfd5aaf74745cf9c8bc1f42c0949988420817d9b297666eb6fa1fbc6ececf6" data-dist-info-metadata="sha256=46866bf88d51d644663ff67a74a13de11229e5096ec2694518a0fa68d116ff94" data-core-metadata="sha256=46866bf88d51d644663ff67a74a13de11229e5096ec2694518a0fa68d116ff94">pyexafs-1.0.0b9-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/80/52/2aa7b2303b78482f2972997c4b2bbeaf49cf2acfd136dc7d5982b59f9f99/pyexafs-1.0.0b9.tar.gz#sha256=645373be13b6765014b2905d3211544582b1b0da0d15a2836b9b80a7441ea460" >pyexafs-1.0.0b9.tar.gz</a><br />
</body>
</html>
<!--SERIAL 23898995-->